backend/src/services/v2/model.ts

```diff
@@ -206,6 +206,20 @@ export async function updateModelCard(
   return revision
 }
 
+export type UpdateModelParams = Pick<ModelInterface, 'name' | 'description' | 'visibility'>
+export async function updateModel(user: UserDoc, modelId: string, diff: Partial<UpdateModelParams>) {
+  const model = await getModelById(user, modelId)
```

Comment:
Why didn't you use `findByIdAndUpdate()`?

Reply:
We need to check the authorisation first.